### ISSUE TITLE
Adding location suggestion 

### ISSUE THREAD

**Issue by Ankesh-erek** [Mohit5Upadhyay/WeatherApplication#26]
I want to add a feature for **location suggestions while typing** in the weather website. I want to work on it so, I'm requesting to assign me this work.

**Comment by Mohit5Upadhyay** [comment]
Yes , @Ankesh-erek  go ahead to implement location suggestion while we enter  location name  as input

**Comment by Ankesh-erek** [comment]
@Mohit5Upadhyay could you kindly add the @GSSOC tag and the appropriate level tag? Thank you!

**Comment by Sawan-Kushwah** [comment]
i want to work ln this issue

**Comment by Ankesh-erek** [comment]
@Sawan-Kushwah I'm already working on it brother and about to done.

**Comment by Mohit5Upadhyay** [comment]
hey @Ankesh-erek the functionality implemented by you is currently not working , please fix them and also the changes also leads to some changes in the ui  , since i seen these changes after merging , so i revert the changes currently , please fix the issue i am happy and  able to merge again this functionality after changes okay bro.

https://github.com/user-attachments/assets/388e81d9-135e-4769-92a4-570b6379dc90

**Comment by Ankesh-erek** [comment]
@Mohit5Upadhyay brother, I'm really sorry for that. I think I may have mistakenly undo some lines of code before merging. I'll definitely fix this issue, but my mid-sem exams are going on, and I'm also working on two projects, so it's been quite hectic for me. Could you please give me some time to resolve it? It would be very helpful. Thank you!

**Comment by Mohit5Upadhyay** [comment]
Ya , @Ankesh-erek  you can take time and implement it accordingly
